pycbc/filter/qtransform.py

```diff
@@ -200,7 +300,7 @@ def qtiling(fseries, qrange, frange, mismatch):
     deltam = deltam_f(mismatch)
     qrange = (float(qrange[0]), float(qrange[1]))
     frange = [float(frange[0]), float(frange[1])]
-    dur = 1.0 / fseries.delta_f
+    dur = fseries.to_timeseries().duration
```

Comment:
same here, don't convert to time domain. In fact, it was OK before, why did you change it?